index.py

```diff
@@ -19,15 +19,15 @@ def filter_punctuations(s):
     Replaces certain punctuations from Strings with space, to be removed later on
     Takes in String s and returns the processed version of it
     """
-    punctuations = '''!?-;:"\\,./#$%^&<>[]{}*_~()'''
+    punctuations = '''!?-;:"\\,./#$%^&<>[]{}*`'=@+…“”’-–—_~()'''
```

Comment:
I dont think we can remove quotation marks as we will need it to detect the phrasal queries. unless we have separate filter functions for index and search

Reply:
The indexing might need to filter cause "keyword ... anotherKeyword" needs to remove the ". In that case, i can add other input for the function that will check if need to remove " and do so accordingly.

Reply:
Yea lets keep it then. I think to make things simpler , instead of checking for the need to remove, we'll just use a separate filter punctuation on the search side.

Reply:
Ill do a pr soon and u can see if its okay

Reply:
Sure thanks

Reply:
its up, if dont want to remove " can use: filter_punctuations(string, True)

Reply:
Hi I think we can still filter the punctuations in the index.py, but it is the search.py that needs to consider the "" phrase queries.

For instance, "Mouse's cheese" would need to be filtered to become Mouse's cheese, but based on our indexing, it should be mapped to Mouse s cheese (assuming we replace apostrophe with space). If we use the apostrophe example, then actually replacing Mouse's with Mouses would actually be better than replacing it with a space. (Mouse VS Mice)